src/System.Drawing.Common/src/System/Drawing/FontFamily.cs

```diff
@@ -185,13 +185,19 @@ private void Dispose(bool disposing)
         /// <summary>
         /// Returns the name of this <see cref='FontFamily'/> in the specified language.
         /// </summary>
-        public string GetName(int language)
+        public unsafe string GetName(int language)
         {
             // LF_FACESIZE is 32
-            var name = new StringBuilder(32);
+            Span<char> name = stackalloc char[32];
```

Comment:
I believe StringBuilder adds a hidden extra element for the trailing null. stackalloc would not, so this is one shorter. Does it matter?

Reply:
Nope I looked in the Windows code and 32 includes the null.